src/commands/fun/blindfoldedroosen.ts

```diff
@@ -0,0 +1,19 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `You've been challenged to a sword fight. However, your opponent, Professor Roosen, has decided to wear a blindfold. Feeling lucky?`;
+	aliases = ['br', 'blindfoldedroosen'];
+
+	run(msg: Message): Promise<Message> {
+		const chosenSword = Math.floor(Math.random() * 6);
+		// 5 is a random number I chose to be the blindfolded Roosen, no other meaning
+		if (chosenSword === 5) {
```

Comment:
In the future I might consider making this and the max number a constant up top instead of the magic number but not a big deal